{
  "uid" : "4b4757e66a1912dae1a509f688f20b0f",
  "name" : "categories",
  "children" : [ {
    "name" : "Test defects",
    "children" : [ {
      "name" : "TypeError: can only concatenate list (not \"str\") to list",
      "children" : [ {
        "name" : "热门文章链接测试",
        "uid" : "391407be9b25ec13",
        "parentUid" : "60817fe824597b25c338e388da83fb11",
        "status" : "broken",
        "time" : {
          "start" : 1636106042005,
          "stop" : 1636106042523,
          "duration" : 518
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "2" ]
      }, {
        "name" : "热门文章链接测试",
        "uid" : "1df6c92e77e6d046",
        "parentUid" : "60817fe824597b25c338e388da83fb11",
        "status" : "broken",
        "time" : {
          "start" : 1636106049238,
          "stop" : 1636106049906,
          "duration" : 668
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "22" ]
      }, {
        "name" : "热门文章链接测试",
        "uid" : "1f761095a02731bc",
        "parentUid" : "60817fe824597b25c338e388da83fb11",
        "status" : "broken",
        "time" : {
          "start" : 1636106104235,
          "stop" : 1636106105830,
          "duration" : 1595
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "14" ]
      } ],
      "uid" : "60817fe824597b25c338e388da83fb11"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=1206): Max retries exceeded with url: /session/98228426e4bf49983f02973fe152a4b2/timeouts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001E70AD61220>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "children" : [ {
        "name" : "热门文章链接测试",
        "uid" : "6ba19c00d0caa6dc",
        "parentUid" : "fa146efbd2fe6698998b3a87e40a34d6",
        "status" : "broken",
        "time" : {
          "start" : 1636106057746,
          "stop" : 1636106057746,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "34" ]
      } ],
      "uid" : "fa146efbd2fe6698998b3a87e40a34d6"
    }, {
      "name" : "urllib3.exceptions.MaxRetryError: HTTPConnectionPool(host='127.0.0.1', port=1207): Max retries exceeded with url: /session/d78e90dbd9913cba749f16f9c14c45ef/timeouts (Caused by NewConnectionError('<urllib3.connection.HTTPConnection object at 0x000001A14B630E50>: Failed to establish a new connection: [WinError 10061] 由于目标计算机积极拒绝，无法连接。'))",
      "children" : [ {
        "name" : "热门文章链接测试",
        "uid" : "4daacae5b6870fb9",
        "parentUid" : "70d6b2ab8c08260893517b57fea2849c",
        "status" : "broken",
        "time" : {
          "start" : 1636106114768,
          "stop" : 1636106114768,
          "duration" : 0
        },
        "flaky" : false,
        "newFailed" : false,
        "parameters" : [ "17" ]
      } ],
      "uid" : "70d6b2ab8c08260893517b57fea2849c"
    } ],
    "uid" : "bdbf199525818fae7a8651db9eafe741"
  } ]
}